notebook/static-src/tree/js/main.js

```diff
@@ -1,46 +1,28 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
-
-require([
-    'jquery',
-    'base/js/namespace',
-    'base/js/dialog',
-    'base/js/events',
-    'base/js/page',
-    'base/js/utils',
-    'services/config',
-    'contents',
-    'tree/js/notebooklist',
-    'tree/js/sessionlist',
-    'tree/js/kernellist',
-    'tree/js/terminallist',
-    'tree/js/newnotebook',
-    'auth/js/loginwidget',
-    // only loaded, not used:
-    'jqueryui',
-    'bootstrap',
-    'custom/custom',
-], function(
-    $,
-    IPython,
-    dialog,
-    events,
-    page,
-    utils,
-    config,
-    contents_service,
-    notebooklist,
-    sesssionlist,
-    kernellist,
-    terminallist,
-    newnotebook,
-    loginwidget){
-    "use strict";
-
+"use strict";
+
+// Contents must be loaded at runtime.
+requirejs(['contents'], function(contents_service) {
+    require('base/js/globals').then(function() {
+    var IPython = require('base/js/namespace');
```

Comment:
Since this is chaining, I'd indent these requires further in.

However, this seems like a weird mish-mash of async requirejs with sync require.